tests/integration/test_route53resolver.py

```diff
@@ -0,0 +1,560 @@
+            SecurityGroupIds=self.get_security_group_ids(ec2_client),
+            Direction="OUTBOUND",
+            Name=resolver_endpoint_name,
+            IpAddresses=self.get_create_resolver_endpoint_ip_address(ec2_client, "18"),
+        )
+
+        create_resolver_endpoint_res = result.get("ResolverEndpoint")
+        resolver_endpoint_id = create_resolver_endpoint_res["Id"]
+
+        # clean up
+        cleanups.append(
+            lambda: route53resolver_client.delete_resolver_endpoint(
+                ResolverEndpointId=resolver_endpoint_id
+            )
+        )
+
+        self._wait_created_endpoint_is_listed_with_status(
+            route53resolver_client, request_id, "OPERATIONAL"
+        )
+        snapshot.match("create_resolver_endpoint_res", create_resolver_endpoint_res)
+
+        rslvr_rule_req_ids = [short_uid(), short_uid(), short_uid()]
+        for ind, req_id in enumerate(rslvr_rule_req_ids):
+            create_resolver_rule_res = route53resolver_client.create_resolver_rule(
+                CreatorRequestId=req_id,
+                RuleType="FORWARD",
+                DomainName=f"www.example{ind}.com",
+                ResolverEndpointId=resolver_endpoint_id,
+                TargetIps=[
+                    {"Ip": "10.0.1.100", "Port": 123},
+                ],
+            )
+
+            create_resolver_rule_res = create_resolver_rule_res.get("ResolverRule")
+            resolver_rule_id = create_resolver_rule_res["Id"]
+            snapshot.match(f"create_resolver_rule_res_{ind}", create_resolver_rule_res)
+
+            delete_resolver_rule = route53resolver_client.delete_resolver_rule(
+                ResolverRuleId=resolver_rule_id
+            )
+            snapshot.match(f"delete_resolver_rule_res{ind}", delete_resolver_rule)
+
+    @pytest.mark.aws_validated
+    def test_delete_non_existent_resolver_rule(self, route53resolver_client, snapshot):
+        resolver_rule_id = "id-123"
+        with pytest.raises(
+            route53resolver_client.exceptions.ResourceNotFoundException
+        ) as resource_not_found:
+            route53resolver_client.delete_resolver_rule(ResolverRuleId=resolver_rule_id)
+        snapshot.match("resource_not_found_res", resource_not_found.value.response)
+
+    @pytest.mark.aws_validated
+    def test_disassociate_non_existent_association(self, route53resolver_client, snapshot):
+        with pytest.raises(
+            route53resolver_client.exceptions.ResourceNotFoundException
+        ) as resource_not_found:
+            route53resolver_client.disassociate_resolver_rule(
+                ResolverRuleId="rslvr-123", VPCId="vpc-123"
+            )
+        snapshot.match("resource_not_found_res", resource_not_found)
+
+    @pytest.mark.aws_validated
+    def test_create_resolver_query_log_config(self, route53resolver_client, cleanups, snapshot):
+        snapshot.add_transformer(snapshot.transform.key_value("Name", "name"))
```

Comment:
```suggestion
        snapshot.add_transformer(snapshot.transform.key_value("Name"))
```

basically the default behavior anyway

Reply:
correct